src/redux/actions/projects/createProject.js

```diff
@@ -11,11 +12,17 @@ const createProject = (
 ) => async (dispatch) => {
   const createdAt = moment().toISOString();
 
+  const newProjectUuid = uuidv4();
+  const newExperimentId = uuidv4();
+
   const newProject = {
     ...projectTemplate,
     name: projectName,
     description: projectDescription,
-    uuid: uuidv4(),
+    uuid: newProjectUuid,
+    // This is only for one sample per experiment
```

Comment:
I don't understand this comment -- we already support multiple sample experiments in the rest of the platform -- did you mean one experiment per project instead?